### PR TITLE
Skip PR tests if only docs are changing

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

**Comment by mehulkar** [comment]
Tested in https://github.com/vercel/turborepo/pull/2266. You can see that JS Package tests ran, but the other unit tests didn't run (because I've temporarily commented out JS package tests in this PR).

CleanShot 2022-10-18 at 14 48 59@2x

**Comment by nathanhammond** [comment]
Can we make this use `turbo-ignore` from in-repo instead? It'll require a bit more (and it isn't a fully-static check) but we should be using our own tools for this.

**Comment by NicholasLYang** [comment]
Unless I'm misunderstanding, this prevents even spinning up an action, which is a lot nicer versus always spinning up an action and early exiting.

**Comment by nathanhammond** [comment]
@NicholasLYang The distinction is real, but it's more-important to dogfood our own code than to precisely optimize based upon path. (Further, hard-coding of paths creates a refactoring hazard that doesn't exist when using `turbo-ignore`.)

See also our internal doc: https://www.notion.so/63e40e5d28ec4d23b2efbe33cfc79537

File: .github/workflows/pr-js-tests.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ on:
     branches: ["main"]
   pull_request:
     types: [opened, edited, synchronize]
+    # paths-ignore:
```
Comment (NicholasLYang) [inline .github/workflows/pr-js-tests.yml:12]: Any reason this is commented out?

**Comment by NicholasLYang** [comment]
Agreed that dogfooding is important but also we're getting **really** slow CI times and it'd be nice to speed them up as much as possible.

**Pull request closed without merging**